gbpservice/nfp/configurator/api/v1/controllers/controller.py

```diff
@@ -96,7 +133,14 @@ def post(self, **body):
             if pecan.request.is_body_readable:
                 body = pecan.request.json_body
 
-            self.rpcclient.cast(self.method_name, body)
+            if self.method_name == 'network_function_event':
```

Comment:
We should not have these if/else cases. Things should happen with a single "route_lookup()" method.